src/ui/flutter_app/lib/screens/game_settings/reset_settings_alert.dart

```diff
@@ -26,7 +26,7 @@ class _ResetSettingsAlert extends StatelessWidget {
   Future<void> _restore(BuildContext context) async {
     Navigator.pop(context);
 
-    // TODO: we should probably enable database deletion in monkey tests
+    // TODO: [Leptopoda] we should probably enable database deletion in monkey tests
```

Comment:
The monkey test does not rely on the developer mode. This was even before I deprecated the DevMode

The mokey test mode can only be accessed with the corresponding compile time flag

Reply:
In the long run, it would be better if the app itself can detect that a high-frequency monkey test is being performed.  Because when the app is uploaded to the Play Store, Google will perform monkey tests on 5 mobile phones. This app often jumps out to the browser before it takes a long time to test, making the second half of the test actually invalid.